astronomer/providers/amazon/aws/triggers/emr.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+from typing import Any, AsyncIterator, Dict, Optional, Tuple
+
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+from astronomer.providers.amazon.aws.hooks.emr import EmrContainerHookAsync
+
+
+class EmrContainerSensorTrigger(BaseTrigger):
+    """
+    EmrContainerSensorTrigger is fired as deferred class with params to run the task in trigger worker
```

Comment:
We should mention when this is fired, for example "This is fired when EMR container is created".

A good example is https://github.com/apache/airflow/blob/cb41d5c02e3c53a24f9dc148e45e696891c347c2/airflow/triggers/temporal.py#L26-L31

Reply:
Fixed as 

```
EmrContainerSensorTrigger is triggered when EMR container is created, it polls for the Job container status.
    It is fired as deferred class with params to run the task in trigger worker
```